src/nvim/eval.c

```diff
@@ -3762,10 +3676,15 @@ int get_option_tv(const char **const arg, typval_T *const rettv, const bool eval
     return OK;
   }
 
+  long numval;
+  char *stringval;
+  bool working = (**arg == '+');  // has("+option")
```

Comment:
this needs to be before `find_option_end(arg, &opt_flags);` above

Reply:
Thanks!